terraform/helm/aptos-node/values.yaml

```diff
@@ -26,16 +26,16 @@ validator:
     pullPolicy: IfNotPresent
   resources:
     limits:
-      cpu: 3.5
-      memory: 6Gi
+      cpu: 15.5
+      memory: 30Gi
     requests:
-      cpu: 3.5
-      memory: 6Gi
+      cpu: 15.5
+      memory: 30Gi
   storage:
     # -- Kubernetes storage class to use for validator persistent storage
     class:
     # -- Size of validator persistent storage
-    size: 350Gi
+    size: 512Gi
```

Comment:
I thought we agreed on 1T?

Reply:
@grao1991 will update this after AIT3. but for the 10 days 512 should be good enough